portfolio/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>YOUR_PROJECT_ID_HERE</application>
+  <application>port-269121</application>
```

Comment:
I don't think its a good idea to be committing project information into the repo as well - you can keep that on your local client when you deploy.

Reply:
Sorry I just started working on the project, I've been terribly busy all week with school and work. What do you mean by this ? Im honestly so confused

Reply:
No worries :) 

The idea here is that this information is a secret that only you need to know. Someone else who looks at this repo doesn't need to know what the project id is (and honestly, probably shouldn't know). The idea here is that you avoid adding configuration files like these into your code submissions so that other people who don't need this information don't see it.